stdlib/public/SDK/Foundation/DateComponents.swift

```diff
@@ -339,9 +339,8 @@ extension DateComponents : _ObjectiveCBridgeable {
     }
 
     public static func _unconditionallyBridgeFromObjectiveC(_ source: NSDateComponents?) -> DateComponents {
-        var result: DateComponents?
-        _forceBridgeFromObjectiveC(source!, result: &result)
-        return result!
+        guard let src = source else { return DateComponents() }
+        return DateComponents(reference: src)
```

Comment:
unrelated, but its a shame how many names we have for the first paraemeter (_bridged / referencing / reference / etc.)